src/js/src/components/SearchSingleItemComponents/SearchSingleItemImages.vue

```diff
@@ -0,0 +1,105 @@
+<template>
+  <div v-if="imageSrcs.length > 0" class="singleEntryImages">
+    <div class="singleEntryImagesHeadline">
+      <p class="highlightText entryInfo imageHeadline">
+        Images:
+      </p>
+      <p> 
+        showing {{ showNumberOfPictures(imageSrcs.length) }} out of {{ imageSrcs.length }}
+      </p>
+      <button v-if="imageSrcs.length > 4" class="allImagesButton" @click="toggleAllImagesShown">
+        {{ shownImagesButtonText }}
```

Comment:
This could be a computed property instead - see my comment above.

Reply:
Fixed.